test/Coverity/README.md

```diff
@@ -0,0 +1,57 @@
+# Static code analysis for FreeRTOS-Plus-TCP library
+This directory is made for the purpose of statically testing the MISRA c-2012 compliance of FreeRTOS+TCP using [Synopsys Coverity](https://www.synopsys.com/software-integrity/security-testing/static-analysis-sast.html) static analysis tool. To that end, this directory provides a [CMake](https://github.com/FreeRTOS/FreeRTOS-Plus-TCP/blob/main/test/Coverity/CMakeLists.txt) file and [configuration files](https://github.com/FreeRTOS/FreeRTOS-Plus-TCP/tree/main/test/Coverity/ConfigFiles) required to build an application for the tool to analyze.
+
+For details regarding the suppressed violations in the report (which can be generated using the following instructions), please see the [MISRA.md](https://github.com/FreeRTOS/FreeRTOS-Plus-TCP/blob/main/MISRA.md) file.
+
+## Getting Started
+### Prerequisites
+You can run this on any platform which is supported by Coverity. The list and other details can be found [here](https://sig-docs.synopsys.com/polaris/topics/c_coverity-compatible-platforms.html).
```

Comment:
on any platform which is supported by Coverity->on a platform supported by Coverity